src/components/Chart.js

```diff
@@ -35,6 +48,8 @@ export default class Chart extends React.Component {
       googleChartLoader.init(this.props.chartPackages, this.props.chartVersion).then(() => {
         this.drawChart();
       });
+      window.addEventListener('resize', this.onResize);
```

Comment:
Could you check if window exists before this line and line 73 so we can continue supporting server-side rendering ?